Using TensorFlow backend.
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/home/vl1019/miniconda3/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/home/vl1019/miniconda3/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/utils/data_utils.py", line 560, in data_generator_task
    generator_output = next(self._generator)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/pescador/maps.py", line 150, in keras_tuples
    for data in stream:
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/pescador/buffered.py", line 82, in iterate
    partial=partial)):
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/pescador/maps.py", line 55, in buffer_stream
    for x in stream:
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/pescador/mux.py", line 213, in iterate
    yield six.advance_iterator(self.streams_[idx])
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/pescador/core.py", line 154, in iterate
    for n, obj in enumerate(self.stream_):
  File "../../../src/054_train-pcen-addntt-convnet.py", line 157, in yield_lms_and_background
    h5py.File(bg_path, "r") as bg_container:
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/h5py/_hl/files.py", line 272, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/h5py/_hl/files.py", line 92, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper (/tmp/pip-tnf92dft-build/h5py/_objects.c:2853)
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper (/tmp/pip-tnf92dft-build/h5py/_objects.c:2811)
  File "h5py/h5f.pyx", line 76, in h5py.h5f.open (/tmp/pip-tnf92dft-build/h5py/h5f.c:2099)
OSError: Unable to open file (Unable to open file: name = '/beegfs/vl1019/birdvox-70k_data/birdvox-70k_clip-logmelspec-backgrounds/t-1800/birdvox-70k_clip-backgrounds_unit01_t-1800.hdf5', errno = 2, error message = 'no such file or directory', flags = 0, o_flags = 0)

2018-02-19 03:13:02.344233 Start.
Training mixture of experts with adaptive threshold on BirdVox-70k with PCEN input. 
Training set: unit07, unit10, unit01.
Validation set: unit02, unit03.
Test set: unit05.

h5py version: 2.6.0
keras version: 2.0.6
numpy version: 1.13.1
pandas version: 0.20.3
pescador version: 1.0.0
tensorflow version: 1.2.1

Traceback (most recent call last):
  File "../../../src/054_train-pcen-addntt-convnet.py", line 371, in <module>
    validation_steps = validation_steps)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/legacy/interfaces.py", line 87, in wrapper
    return func(*args, **kwargs)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/engine/training.py", line 1809, in fit_generator
    generator_output = next(output_generator)
StopIteration
